functional_testing/zrobot_templates/framework/ovc_utils/utils.py

```diff
@@ -75,7 +75,13 @@ def get_portforward_list(self, cloudspacename, machinename):
         cloudspaceId = self.get_cloudspace(cloudspacename)['id']
         machineId = self.get_vm(cloudspaceId, machinename)['id']
         return self.ovc_client.api.cloudapi.portforwarding.list(cloudspaceId=cloudspaceId, machineId=machineId)
-    
+
+    def get_snapshots_list(self, cloudspacename, machinename):
+        time.sleep(2)
```

Comment:
please remove time.sleep from here and from all methods as long as it is not needed .. i feel we should do the time sleep in the test case only when needed